apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -1317,6 +1324,20 @@ export const extensions: Extensions = {
                             "application that should not be secured, and do not require authentication. Multiple " +
                             "URIs can be set using <3>comma separated</3> values."
                     }
+                },
+                m2m: {
+                    configurations: {
+                        clientId: {
+                            hint: "The Client ID generated when registering the application in Asgardeo."
```

Comment:
Lets define a placeholder for the product name and pass the product name through i18n options.

Please check and fix the other places as well.

```suggestion
                            hint: "The Client ID generated when registering the application in {prpductName}."
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5850/commits/3b40c638a941e99c8405c5a1650041f3e872d3c5